src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v1/SwitchController.java

```diff
@@ -254,18 +254,17 @@ public CompletableFuture<List<Long>> installSwitchRules(
      *      - AUTO - this is the default. Installs all default rules when a switch connects
      *      - SAFE - add the default rules slowly .. monitoring traffic on existing rules
      *      - MANUAL - don't install any default rules. Call addRule for that.
-     * NOTE: no action is taking with existing, connected switches. This operation will only affect
-     *      future connections
+     * NOTE: not supported anymore.
      *
      * @param mode the connectMode to use. A Null value is a No-Op and can be used to return existing value.
      * @return the value of the toggle in Floodlight.
      */
-    @ApiOperation(value = "Set the connect mode if mode is specified. If mode is null, this is effectively a get.",
-            response = ConnectModeRequest.Mode.class)
+    @Deprecated
+    @ApiOperation(value = "Connect mode is not supported anymore.", response = ConnectModeRequest.Mode.class)
     @PutMapping(value = "/toggle-connect-mode")
     @ResponseStatus(HttpStatus.OK)
     public CompletableFuture<Mode> toggleSwitchConnectMode(@RequestParam("mode") ConnectModeRequest.Mode mode) {
-        return switchService.connectMode(mode);
+        return CompletableFuture.completedFuture(Mode.AUTO);
```

Comment:
Can we return an error instead of faked result? E.g. 410 http response?

Reply:
+1

Reply:
Changed.